packages/draw/src/artist.ts

```diff
@@ -2,17 +2,42 @@ import { Canvas, Drawable } from './canvas';
 import { Zoomer } from './zoomer';
 import { Styles, createStyles, isSSR } from './style';
 import { GraphStyleConfig, Focus } from './types';
+import { Animation, AnimationConfig } from './animation';
+import { LayerTransition, TransitionManager } from './transition';
 
 export interface ArtistArgs {
   style?: Partial<GraphStyleConfig>;
   canvas: HTMLCanvasElement;
 }
 
-interface Layer {
+export interface Layer {
   drawables: Drawable[];
   focus: Focus;
+  animation?: Animation<number>;
 }
 
+const DIMMING_ANIMATION_CONFIG: (styles: Styles) => AnimationConfig<number> = (
+  styles: Styles,
+) => {
+  return {
+    duration: 1,
```

Comment:
I probably need to externalize the duration of our animations and transitions to `styles`

Reply:
I ended up doing this